Changelog.md

```diff
@@ -3,6 +3,7 @@
 Language Features:
  * General: Support ``ContractName.functionName`` for ``abi.encodeCall``, in addition to external function pointers.
  * General: Add equality-comparison operators for external function types.
+ * General: Allow annotating inline assembly as memory-safe to allow optimizations and stack limit evasion that rely on respecting Solidity's memory model.
```

Comment:
This only applies to "via ir", does it?

Reply:
In practice at least... in principle, we could use the information anywhere, but we probably won't.